package.json

```
@@ -17,22 +20,40 @@
   ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/blackjk3/react-signature-pad.git"
+    "url": "https://github.com/martenwallewein/react-signature-pad.git"
   },
-  "author": "Jason Kadrmas",
+  "author": "Marten Wallewein",
```

Ich würde hier zumindest Kadrmas mit rein schreiben. Ist ja immerhin ein Fork.
```suggestion
  "author": "Marten Wallewein, Jason Kadrmas",
```